client/my-sites/site-settings/jetpack-credentials/index.jsx

```diff
@@ -60,7 +60,7 @@ class Backups extends Component {
 				<QueryJetpackCredentials siteId={ this.props.siteId } />
 				{ isRewindActive && (
 					<CompactCard className="jetpack-credentials__header">
-						<span>{ translate( 'Backups and restores' ) }</span>
+						<span>{ translate( 'Backups and security scans' ) }</span>
```

Comment:
Hi! I've found a possible matching string that has already been translated **29** times:
`translate( 'Security scans &amp; backups' )` *ES Score: 12*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/remove/restore)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).